lms/static/js/student_account/views/FormView.js

```diff
@@ -69,6 +69,51 @@
                 this.$form = $container.find('form');
                 this.$errors = $container.find('.submission-error');
                 this.$submitButton = $container.find(this.submitButton);
+                this.setUserCityAndCountry();
+                this.showPassword();
+                this.getUsernameFromEmail();
+                this.styleShowPassword();
+            },
+
+            styleShowPassword: function() {
+                $('input#register-show_password').css('display', 'none');
+                $('div.form-field.checkbox-show_password label').css({
+                  'font-size': '0.8rem',
+                  'color': '#27aae1',
+                  'float': 'right'
+                });
+                $('.form-field.checkbox-show_password').css({
+                  'margin-top': '-20px',
+                  'top': '-50px'
+                });
+            },
```

Comment:
Those form fields are rendered after page load and not on page load by underscore, I guess! I once tried targeting them in scss, but didn't work. Please view the page source of the register page to confirm.